docs/sources/installation/ubuntulinux.md

```diff
@@ -86,15 +86,23 @@ This installation path should work at all times.
 
 ### Dependencies
 
-**Linux kernel 3.8**
-
-Due to a bug in LXC, Docker works best on the 3.8 kernel. Precise comes
-with a 3.2 kernel, so we need to upgrade it. The kernel you'll install
-when following these steps comes with AUFS built in. We also include the
-generic headers to enable packages that depend on them, like ZFS and the
-VirtualBox guest additions. If you didn't install the headers for your
-"precise" kernel, then you can skip these headers for the "raring"
-kernel. But it is safer to include them if you're not sure.
+**Linux kernel 3.13**
+
+Kernel 3.13 is currently the recommended kernel version for Ubuntu Precise.
+Some Ubuntu Precise installs have an older kernel installed, so it must
+be upgraded. The kernel you'll install when following these steps has AUFS
+built in.
+We also include the generic headers to enable packages that depend on them,
+like ZFS and the VirtualBox guest additions. If you didn't install the
+headers for your "precise" kernel, then you can skip these headers for the
+"trusty" kernel. It is safer to include the headers if you're not sure.
```

Comment:
Eliminate passive voice: If you're unsure, you should include the headers for safety.